src/libraries/System.Security.Claims/src/System/Security/Claims/GenericIdentity.cs

```diff
@@ -9,8 +9,8 @@ namespace System.Security.Principal
 {
     public class GenericIdentity : ClaimsIdentity
     {
-        private readonly string m_name;
-        private readonly string m_type;
+        private readonly string m_name = null!;  // Init-ed in parameterized constructors
+        private readonly string m_type = null!;
```

Comment:
I'm assuming this is needed because of the default constructor. Is that being used anywhere?

Reply:
Yes for default constructor, that is private and not used anywhere, I could remove that instead but normally I avoid such change unless others ok with that

Reply:
Agreed, we shouldn't be removing the constructor.

Reply:
Why wouldn't we remove it?  What's using it?  It's a private member, there are other ctors so a default one won't be added by the compiler, and it appears it's not used anywhere.

Reply:
For me I hesitate remove it because there can be a reason which I don't know, which made them add this unused private empty constructor. As I said I am ok with removing it, will remove then .